### ISSUE TITLE
MISOC Logging

### ISSUE THREAD

**Issue by ccoffrin** [mlubin/Pajarito.jl#179]
In this case with log_level=0, ` PajaritoSolver(mip_solver=GLPKSolverMIP(), cont_solver=scs_solver, log_level=0)`, I still get this output,
```
Transforming original data......Done   0.56s

Creating conic model data......Done   0.31s

Creating MIP model......Done   0.66s

Solving conic relaxation......Done   0.22s

Creating conic subproblem model......Done   0.00s

Starting iterative outer approximation algorithm

Pajarito MICP solve summary:
 - Total time (s)       =       2.40e+00
 - Status               =     Suboptimal
 - Best feasible obj.   =  -4.287657e+00
 - Final OA obj. bound  =  -4.288606e+00
 - Relative opt. gap    =      2.214e-04
```

**Comment by chriscoey** [comment]
OK thanks, I've changed this and amended the readme.

**Comment by chriscoey** [comment]
Will make a PR soon.

**Comment by chriscoey** [comment]
closed by 431a4db
